### PR TITLE
New transport to implement fixed latency between pairs of peers

### PR DESCRIPTION
**Pull request by oascigil**

### CODE REVIEW

File: simulator/src/main/java/peersim/transport/PairwiseFixedLatencyTransport.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package peersim.transport;
+
+import peersim.config.*;
+import peersim.core.*;
+import peersim.edsim.*;
+
+public final class PairwiseFixedLatencyTransport extends UniformRandomTransport {
+
+  /** Latency between nodes (symmetric). */
+  private static int[][] array;
```
Comment (srene) [inline simulator/src/main/java/peersim/transport/PairwiseFixedLatencyTransport.java:10]: i think this is not used, so can be removed

File: simulator/src/main/java/peersim/transport/PairwiseFixedLatencyTransport.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    long delay = getLatency(src, dest);
+
+    EDSimulator.add(delay, msg, dest, pid);
+  }
+
+  /**
+   * Returns the assigned delay to the specific src and dest peers that was previously generated.
+   */
+  public long getLatency(Node src, Node dest) {
+    int sender = src.getIndex() % size;
```
Comment (srene) [inline simulator/src/main/java/peersim/transport/PairwiseFixedLatencyTransport.java:52]: according to the docs, getIndex() can return different values during simulation, so it is better to use getId()

File: simulator/src/main/java/peersim/transport/PairwiseFixedLatencyTransport.java (hunk 1 above)
Comment (oascigil) [reply simulator/src/main/java/peersim/transport/PairwiseFixedLatencyTransport.java]: yes, removed it.

File: simulator/src/main/java/peersim/transport/PairwiseFixedLatencyTransport.java (hunk 2 above)
Comment (oascigil) [reply simulator/src/main/java/peersim/transport/PairwiseFixedLatencyTransport.java]: Fixed this as well. Thanks!

**Review by srene — APPROVED** [review]

**Pull request merged**